views/components/icons/index.jsx

```diff
@@ -0,0 +1,17 @@
+import { createElement, safe } from 'complate-stream'
+
+function ArrowRightDef () {
+  return <svg id="icon-arrow-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>{`.cls-1{fill:none;stroke:currentColor;stroke-linecap:round;stroke-linejoin:round;stroke-width:1.5px;fill-rule:evenodd;}`}</style></defs><title>arrow-right-1</title><path class="cls-1" d="M5.5.75,16.22,11.47a.749.749,0,0,1,0,1.06L5.5,23.25"/></svg>;
```

Comment:
FWIW: https://stackoverflow.com/a/18468348

Reply:
👍 I will look at that.

My current process was to just copy the SVG directly from the .svg file into my macro, but I think it might be a good idea to do a few modifications (like moving the inline styles into our sass files).